<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbCmdPadroes">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>*IDN?</string>
     </property>
    </item>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>581</width>
      <height>271</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="pagComandos">
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>&amp;Comandos</string>
     </attribute>
     <widget class="QTreeWidget" name="tvwComandos">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>71</y>
        <width>541</width>
        <height>171</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
     <widget class="QComboBox" name="cbDispositivos">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>541</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDetectar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>6</y>
        <width>541</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;Detectar dispositivos</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>img/update.png</normaloff>img/update.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>&amp;Configurações</string>
     </attribute>
     <widget class="QComboBox" name="cbPortaComm">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>GPIB0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GPIB1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GPIB2</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="lblPorta">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Porta de comunicação</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblLargura_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Altura da Tela:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbLargura">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>1280</number>
      </property>
      <property name="value">
       <number>640</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbAltura">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>1024</number>
      </property>
      <property name="value">
       <number>480</number>
      </property>
     </widget>
     <widget class="QLabel" name="lblLargura">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Largura da Tela:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbLerEnviar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>571</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <widget class="QTextEdit" name="txtRead">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLer">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>86</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Ler</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtSend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>451</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEnviar">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>86</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Enviar</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>71</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowGraphics">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>340</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exibe Gráfico</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>755</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDetect_Devices">
   <property name="text">
    <string>Detect Devices</string>
   </property>
  </action>
  <action name="actionRead">
   <property name="text">
    <string>Read</string>
   </property>
  </action>
  <action name="actionWrite">
   <property name="text">
    <string>Write</string>
   </property>
  </action>
  <action name="action_StandardCommands">
   <property name="text">
    <string> Standard Commands</string>
   </property>
  </action>
  <action name="actionShowGraphics">
   <property name="text">
    <string>ShowGraphics</string>
   </property>
  </action>
  <action name="actionCmdPadroesChanged">
   <property name="text">
    <string>CmdPadroes</string>
   </property>
  </action>
  <action name="actionDeviceSelected">
   <property name="text">
    <string>DeviceSelected</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnDetectar</sender>
   <signal>clicked()</signal>
   <receiver>actionDetect_Devices</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEnviar</sender>
   <signal>clicked()</signal>
   <receiver>actionWrite</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLer</sender>
   <signal>clicked()</signal>
   <receiver>actionRead</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnShowGraphics</sender>
   <signal>clicked()</signal>
   <receiver>actionShowGraphics</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>675</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbCmdPadroes</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>actionCmdPadroesChanged</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbDispositivos</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>actionDeviceSelected</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
